docs/hubs-discord-bot.md

```diff
@@ -4,44 +4,68 @@ title: Discord Bot
+
 
 
 ### Bot Permissions
+
 The bot requires several permissions in order to work. 
-* "Send messages," "Read messages," and "Embed links" are necessary in order to bridge between the Hubs room that is linked to a channel and the messages that are sent within the channel on Discord. 
-* "Manage webhooks" is necessary in order for the bot to find and use a webhook for bridging chat. 
-* "Manage channels" is necessary in order for the bot to set the channel topic and bridge chat. Note: We do not ask for this permission globally when you add the bot to your server, instead we recommend you grant this permission to the bot in specific groups or channels. 
+* "Send messages" and "Embed Links" are necessary in order to bridge between the Hubs room that is linked to a channel and the messages that are sent within the channel on Discord.
+* "Manage Webhooks" is necessary in order for the bot to either create or find and use a webhook for bridging chat.
+* "Manage Channels" is necessary in order for the bot to set the channel topic and bridge chat.
+* "Read Message History" and "Manage Messages" are needed in order for the bot to pin notification messages.
+* "Mention @everyone, @here, and All Roles" is necessary in order for the bot to notify the people present when the scheduled notification messages are posted.
+
+Note: We advise that instances of the bot do not ask for the "Manage Channels", "Manage Messages", "Read Message History", and "Mention @everyone, @here, and All Roles", permissions globally when you add the bot to your server, instead we recommend you grant this permission to the bot in specific groups or channels.
+
+### Creating a Bound Hubs Room
+
+Use the `!hubs create` command (optionally with a scene URL and a name) to create a room with permissions that are bound to Discord.  The room URL will be placed in the channel topic and so bridge the chat as well.
+
+Note: There is currently no way to unbind a Hubs room from Discord.  If you no longer want a channel to be bridged to a bound room you will need to create a new regular Hubs room and bridge that unbound room to the channel.
+
+### Bridging a Hubs Room
+
+To bridge the chat between a Discord channel and a Hubs room, simply add the URL of the room to the channel topic to connect the bot to the room and have it start bridging stuff between Hubs and Discord.  Removing the URL from the channel topic will unbridge the room.
+
+Note: Bridging the room won't bind the room to Discord if it's not already a bound room.
 
 ### Commands
 
-The following features outline the current bot commands and how they can be accessed by users in the Discord server:
+The following features outline the current bot commands and how they can be accessed by users in the Discord server (in order to use a command, simply send a message that only contains the command in it):
```

Comment:
"only contains" or "starts with"?

Reply:
Adding additional content to a command message may cause the command not to work, so I think it's better to go with "only contains".